src/TicTacToe/EndingConditions.java

```diff
@@ -37,8 +37,15 @@ boolean isDraw() {
         return true;
     }
 
-    boolean isWin() {
+    boolean isWin(){
+        if (isWinDiagonally()||isWinHorizontally()||isWinVertically()){
```

Comment:
You can do this method in one line just:
return isWinDiagonally()||isWinHorizontally()||isWinVertically();